script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -94,30 +94,30 @@ To be honest, I couldn't stand that smile anymore.
 "What is it, Shiki-kun?"
 <Page26>
 "Why do you have lines all over your body?
-And there are so many cracks in this room as well, it looks like it's about to fall apart."
+And there are so many cracks in this room, it looks like it's about to fall apart."
 <Page27>
 Although his smile vanished for a brief moment, he quickly returned to his smiling face and stood up from his chair.
 The doctor clattered across the floor and vanished behind a curtain.
 <Page28>
-"...It seems there was some brain damage after all.
+"...It seems there is some brain damage after all.
 Contact Doctor Ashiya in the neurosurgery department. I also suspect some damage to the optic nerves. Send him in for an eye examination this afternoon."
 The doctor whispered to the nurse in a cold voice, as if he was an entirely different person.
 <Page29>
-"....It's weird. There are lines all over everyone's bodies."
+"......It's weird. There are lines all over everyone's bodies."
 <Page30>
 The messy lines were all over the hospital.
-I didn't know what they meant, but I felt sick just looking at them.
+I didn't know what they meant, but I felt sick just by looking at them.
 <Page31>
 "...What are these things?"
-The lines were on the bed too.
-When I touched it with my finger, #Poke. #My finger sank in.
+The lines were on this bed too.
+When I touched it with my finger, #poke, #my finger sank in.
```

Comment:
'the bed' reads better to me.

 I'm not sure how the second line is formatted but original line is better. Capitalization of Poke should be addressed though, maybe replace the comma with an ellipses. Also, should be changed to 'When I touched **one of them** with my finger'